src/build.ts

```diff
@@ -10,73 +15,22 @@ export const validateAndCreateFunctions = async (
   manifest: any,
   protocol: Protocol,
 ) => {
-  // Ensure functions directory exists
+  // Ensure functions output directory exists
   const functionsPath = path.join(outputDirectory, "functions");
   await ensureDir(functionsPath);
 
-  // Find all the run on slack functions
```

Comment:
All of the validation-type work in here is now moved to the new utility method in `utilities.ts`.